src/admin-views/tribe-options-display.php

```diff
@@ -3,28 +3,129 @@
-if ( $should_default_to_on ) {
+$etp_active = $dep->is_plugin_active( 'Tribe__Tickets_Plus__Main' );
+
+// Don't display this section if ET is a newer install and ETP is inactive or ETP is active but a newer install.
+if (
+	(
+		! $etp_active
+		&& ! tribe_installed_before( 'Tribe__Tickets__Main', '5.0' )
+	)
+	|| (
+		$etp_active
+		&& ! tribe_installed_before( 'Tribe__Tickets_Plus__Main', '5.1' )
+	)
+) {
 	return;
 }
 
-$settings = Tribe__Main::array_insert_before_key( 'tribe-form-content-end', $settings, [
-	'rsvp-display-title'         => [
+$tickets_rsvp_display_title = sprintf(
+	esc_html_x(
+		'%1$s Display Settings',
+		'title of settings section',
+		'event-tickets'
+	),
+	tribe_get_rsvp_label_plural( 'tickets_rsvp_display_title' )
+);
+
+$tickets_rsvp_display_description = sprintf(
+	esc_html_x(
+		'The settings below control the display of your %1$s.',
+		'description of settings section',
+		'event-tickets'
+	),
+	tribe_get_rsvp_label_plural( 'tickets_rsvp_display_description' )
+);
+
+if ( $etp_active ) {
+	$tickets_rsvp_display_title = sprintf(
+		esc_html_x(
+			'%1$s and %2$s Display Settings',
+			'title of settings section',
+			'event-tickets'
+		),
+		tribe_get_ticket_label_plural( 'tickets_rsvp_display_title' ),
+		tribe_get_rsvp_label_plural( 'tickets_rsvp_display_title' )
+	);
+
+	$tickets_rsvp_display_description = sprintf(
+		esc_html_x(
+			'The settings below control the display of your %1$s and %2$s.',
+			'description of settings section',
+			'event-tickets'
+		),
+		tribe_get_ticket_label_plural( 'tickets_rsvp_display_description' ),
+		tribe_get_rsvp_label_plural( 'tickets_rsvp_display_description' )
+	);
+}
+
+$et_options_display = [
+	'tickets_rsvp_display_title'       => [
 		'type' => 'html',
-		'html' => '<h3>' . __( 'RSVP Display Settings', 'event-tickets' ) . '</h3>',
+		'html' => '<h3 id="tickets_rsvp_display_title">' . $tickets_rsvp_display_title . '</h3>',
 	],
-	'rsvp-display-description'   => [
+	'tickets_rsvp_display_description' => [
 		'type' => 'html',
-		'html' => '<p>' . __( 'The settings below control the display of your RSVPs.', 'event-tickets' ) . '</p>',
+		'html' => '<p>' . $tickets_rsvp_display_description . '</p>',
 	],
-	'tickets_rsvp_use_new_views' => [
+];
+
+// Only show this option to older installs, as newer installs default to new views.
+if ( tribe_installed_before( 'Tribe__Tickets__Main', '5.0' ) ) {
+	$et_options_display['tickets_rsvp_use_new_views'] = [
 		'type'            => 'checkbox_bool',
-		'label'           => __( 'Enable New RSVP Experience', 'event-tickets' ),
-		'tooltip'         => __( 'This setting will render the new front-end designs (styling) and user-flow for the RSVP experience.', 'event-tickets' ),
+		'label'           => sprintf(
+			esc_html_x(
+				'Enable New %1$s Experience',
+				'settings label',
+				'event-tickets'
+			),
+			tribe_get_rsvp_label_singular( 'tickets_rsvp_use_new_views' )
+		),
+		'tooltip'         => sprintf(
+			esc_html_x(
```

Comment:
:no_entry_sign: A gettext call containing placeholders was found, but was not accompanied by a "translators:" comment on the line above to clarify the meaning of the placeholders.

<sub>hash: 8e3eb81607681044a22a7220f56e8ded</sub>